### ISSUE TITLE
Booster Algorithms Are Missing

### ISSUE THREAD

**Issue by gaur-avvv** [kamranahmedse/developer-roadmap#9342]
### Roadmap URL

https://roadmap.sh/machine-learning

### Suggestions

Here is the list of boosting algorithms which are missing:

GradientBoosting 
AdaBoost 
XGBoost 
CatBoost 
LightGBM

**Comment by jcanalesluna** [comment]
This is fixed now, We add more details on booster algorithms in the Gradient Boosting Node
